packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/__tests__/quanticDocumentSuggestion.test.js

```diff
@@ -0,0 +1,371 @@
+            .getAttribute('data-doc-id')
+        ).toBe(uniqueId);
+      });
+    });
+
+    it('should not render quick view button when withoutQuickview is set to true', async () => {
+      const element = createTestComponent({
+        withoutQuickview: true,
+      });
+      await updateSuggestions();
+
+      expect(element.shadowRoot.querySelector(selectors.quickview)).toBeFalsy();
+    });
+
+    it('should log a suggestion click if not opened', async () => {
+      const element = createTestComponent();
+      await updateSuggestions();
+
+      const sections = element.shadowRoot.querySelectorAll(
+        selectors.accordionSection
+      );
+      const accordion = element.shadowRoot.querySelector(selectors.accordion);
+
+      // First document is already opened and will be collapsed, no click event logged.
+      sections[0].click();
+      expect(functionsMocks.logDocumentSuggestionClick).toHaveBeenCalledTimes(
+        0
+      );
+      accordion.activeSectionName = sections[1].name;
+
+      sections[1].click();
+      await flushPromises();
+      expect(functionsMocks.logDocumentSuggestionClick).toHaveBeenCalledTimes(
+        1
+      );
+      expect(functionsMocks.logDocumentSuggestionClick).toHaveBeenCalledWith(
+        documentSuggestionListMock.state.documents[1].uniqueId
+      );
+    });
+
+    it('should handle document rating event', async () => {
+      const element = createTestComponent();
+      await updateSuggestions();
+
+      const customEvent = new CustomEvent('quantic__rating', {
+        detail: {id: 'mastercraft', score: 2},
+      });
+      element.shadowRoot.dispatchEvent(customEvent);
+      expect(functionsMocks.logDocumentSuggestionRating).toHaveBeenCalledTimes(
+        1
+      );
+      expect(functionsMocks.logDocumentSuggestionRating).toHaveBeenCalledWith(
+        'mastercraft',
+        2
+      );
+    });
+
+    it('should truncate documents to maxDocuments if the state contains more', async () => {
+      const element = createTestComponent({
+        maxDocuments: 1,
+      });
+      await updateSuggestions();
+
+      const sections = element.shadowRoot.querySelectorAll(
+        selectors.accordionSection
+      );
+      expect(sections.length).toBe(1);
+    });
+
+    it('should open the documents depending on numberOfAutoOpenedDocuments', async () => {
+      const element = createTestComponent({
+        fetchOnInit: true,
+        numberOfAutoOpenedDocuments: 2,
+      });
+      await updateSuggestions();
+
+      const accordion = element.shadowRoot.querySelector(selectors.accordion);
+      const sections = element.shadowRoot.querySelectorAll(
+        selectors.accordionSection
+      );
+      expect(sections.length).toBe(2);
+      expect(accordion.activeSectionName).toEqual([
+        sections[0].name,
+        sections[1].name,
+      ]);
+    });
+  });
+
+  describe('when two document suggestions have a similar title', () => {
```

Comment:
the cypress test do this with a component that has `numberOfAutoOpenedDocuments` set to 0:
https://github.com/coveo/ui-kit/blob/5946a9eb6c8f0f69848fa0565fc3b1dea8ea55fb/packages/quantic/cypress/e2e/default-2/document-suggestion/document-suggestion.cypress.ts#L210-L215

I think we should do the same here, as this cypress test was added after a bug that we fixed, so we need to replicate exactly the stup that was causing the issue previously.